mycity/mycity/deploy_tools/deploy_tools.py

```diff
@@ -1,24 +1,26 @@
-'''
+"""
 Tools to package and deploy the lambda function for Boston Data app.
-'''
+"""
 
 from __future__ import print_function
+from subprocess import run
 import argparse
 import os
-import pip
 import shutil
-import subprocess
 import zipfile
+import stat
+import errno
 
-# TODO: This needs to be updated to give user some idea of how to run it and where to run it from
+# path constants
 PROJECT_ROOT = os.path.join(os.getcwd(), os.pardir, os.pardir)
 TEMP_DIR_PATH = os.path.join(PROJECT_ROOT, 'temp')
 LAMBDA_REL_PATH = 'platforms/amazon/lambda/custom/lambda_function.py'
 LAMBDA_FUNCTION_PATH = os.path.join(PROJECT_ROOT, LAMBDA_REL_PATH)
 MYCITY_PATH = os.path.join(PROJECT_ROOT, 'mycity')
-
 INSTALL_REQUIREMENTS_SCRIPT = os.path.join(os.getcwd(), 'install_requirements.sh')

```

Comment:
Can we get rid of this constant and the `install_requirements.sh`

Reply:
Yes, definitely. Will do that and update when I have a chance.